pyproject.toml

```diff
@@ -1,3 +1,3 @@
 [build-system]
-requires = ["setuptools", "wheel", "setuptools_scm"]
+requires = ["setuptools>=42", "wheel", "setuptools_scm[toml]>=4.1.0"]
```

Comment:
@dihm What's the reason for the toml requirement?

Seems to me that it shouldn't be necessary since setuptools_scm only uses toml to read its configuration from pyproject.toml. Since we still need to set its config in setup.py based on an environment variable, we aren't using that.

Reply:
Ah, looks like you were probably copying from the recommended usage here: https://pypi.org/project/setuptools-scm/

including the requirement to use setuptools > 42.

I'm going to leave these changes out for simplicity.